.travis.yml

```diff
@@ -1,15 +1,20 @@
 sudo: false
 language: node_js
 node_js:
-  - "5.5"
+- '5.5'
```

Comment:
I'd change that to '6'. First of all, Node 5 is EOL'd in about a month, secondly, we shouldn't pin to a minor as only the latest n.x.y for each n is supported by upstream.